tests/unit_tests/admin/test_admin.py

```
@@ -53,18 +53,18 @@ def test_admin_panel_handles_empty_plugins_data(mock_render_template, admin_blue
 
 @patch("platzky.admin.admin.render_template")
-def module_settings_renders_module_when_user_exists(mock_render_template, admin_blueprint):
+def test_module_settings_renders_module_when_user_exists(mock_render_template, admin_blueprint):
     with admin_blueprint.test_request_context("/admin/module/test_module"):
         session["user"] = "test_user"
-        admin_blueprint.view_functions["admin.module_settings"]()
+        admin_blueprint.view_functions["admin.module_settings"]("test_module")
         mock_render_template.assert_called_with(
             "module.html", user="test_user", module_name="test_module"
-        )
+        )
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)